fv3core/utils/grid.py

```diff
@@ -453,14 +455,23 @@ def lat(self) -> FloatFieldIJ:
         raise NotImplementedError()
 
 
-@dataclasses.dataclass(frozen=True)
+@dataclasses.dataclass
 class VerticalGridData:
     """
     Terms defining the vertical grid.
 
     Eulerian vertical grid is defined by p = ak + bk * p_ref
     """
 
+    # TODO: make these non-optional, make FloatFieldK a true type and use it
```

Comment:
No, but we will see this TODO when we go to use Oliver's work to populate these data classes. These can't be made non-optional right now because they are not stored in the grid savepoint.

Reply:
The new style I'd like try and adopt is `TODO(jeremym):  make these non-optional, make FloatFieldK a true type and use it` so that we know who to ask if someone later has questions about it (and doesn't want to use `git blame`).